docs/src/CommutativeAlgebra/module_localizations.md

```diff
@@ -0,0 +1,81 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["module_localizations.md"]
+```
+
+# Localizations of modules over computable rings
+
+For localizations of modules, there exists a generic implementation of 
+the common methods such as membership tests, kernel computations, etc. 
+based on the work of Barakat, Posur, et. al; see [Pos18](@cite).
+
+Let $R$ be a ring of type `<:Ring`, $U \subset R$ a multiplicative set of type `<:AbsMultSet` 
```

Comment:
Is it really necessary to say `of type `<:Ring` ? Isn't this always implied? OTOH I would say explicitly that the ring must be commutative.

Same for the other type: this is useful in methods, but aren't we describing the mathematics here?


```suggestion
Let $R$ be a commutative ring, $U \subset R$ a multiplicative set 
```